### PR TITLE
Bug 1524521 - Provisioning local perf data is hard

### PR DESCRIPTION
**Pull request by ionutgoldan**
The workers were easily breaking because of badly allocating the tasks + when accessing secured data.
This PR addresses both these problems.

### CODE REVIEW

File: treeherder/perf/management/commands/import_perf_data.py [hunk 1]
```diff
@@ -197,8 +198,14 @@ def save_local_data(self):
             print('Saving {0} data...'.format(table_name))
             model_values = properties['model'].objects.using(self.source).filter(
                 pk__in=self.models_instances[table_name])
+            self._ignore_sensitive_fields(table_name, model_values)
             properties['model'].objects.using(self.target).bulk_create(model_values)
 
+    def _ignore_sensitive_fields(self, table_name, model_values):
+        if table_name == 'performance_alert':
+            for model in model_values:
+                model.classifier = None
```
Comment (armenzg) [inline treeherder/perf/management/commands/import_perf_data.py]: Why setting classifier to None ignores some fields?

File: treeherder/perf/management/commands/import_perf_data.py [hunk 2]
```diff
@@ -197,8 +198,14 @@ def save_local_data(self):
             print('Saving {0} data...'.format(table_name))
             model_values = properties['model'].objects.using(self.source).filter(
                 pk__in=self.models_instances[table_name])
+            self._ignore_sensitive_fields(table_name, model_values)
             properties['model'].objects.using(self.target).bulk_create(model_values)
 
+    def _ignore_sensitive_fields(self, table_name, model_values):
```
Comment (armenzg) [inline treeherder/perf/management/commands/import_perf_data.py]: Could you please add a note for the function as to why we need to ignore some fields?

File: treeherder/perf/management/commands/import_perf_data.py (hunk 1 above)
Comment (ionutgoldan) [reply treeherder/perf/management/commands/import_perf_data.py]: > Why setting classifier to None ignores some fields?

The UPSTREAM_DATABASE_URL credentials I'm using don't have enough privilege to read the `auth_user` table. Thus, trying to set the `classifier` field ends up in a runtime error.

File: treeherder/perf/management/commands/import_perf_data.py (hunk 2 above)
Comment (ionutgoldan) [reply treeherder/perf/management/commands/import_perf_data.py]: Sure thing.

**Pull request merged**